playwright-dependencies/tests/smoke.spec.ts

```diff
@@ -0,0 +1,42 @@
+import { test, expect } from '@playwright/test';
+test('smokeTest', async ({ page }) => {
+  const url = "localhost:" + process.env.CONTAINER_EXPORTED_PORT;
+
+  await page.goto(url);
+
+  await page.getByRole('button', { name: 'Sign In' }).click();
+  await page.getByLabel('Email Address').click();
+  await page.getByLabel('Email Address').fill('test@liferay.com');
+  await page.getByLabel('Email Address').press('Tab');
+  await page.getByLabel('Password').fill('test');
+  await page.getByLabel('Password').press('Tab');
+  await page.getByLabel('Remember Me').press('Tab');
+  await page.getByLabel('Sign In- Loading').getByRole('button', { name: 'Sign In' }).press('Enter');
+
+  const newPasswordField = page.locator(".float-right");
+  await newPasswordField.waitFor({state: "visible"});
+
+  await page.getByLabel('Password', { exact: true }).click();
+  await page.getByLabel('Password', { exact: true }).fill('new-password');
+  await page.getByLabel('Password', { exact: true }).press('Tab');
+  await page.getByLabel('Reenter Password').fill('new-password');
+  await page.getByLabel('Reenter Password').press('Tab');
+  await page.getByRole('button', { name: 'Save' }).press('Enter');
+
+  const responsePromise = page.waitForResponse('**/o/com-liferay-enterprise-product-notification-web/confirm/');
+  await page.getByRole('button', { name: 'Done' }).click();
+  const response = await responsePromise;
```

Comment:
@lucasmiranda0, can we do the following?
```
await page.waitForResponse('**/o/com-liferay-enterprise-product-notification-web/confirm/');
await page.getByRole('button', { name: 'Done' }).click();
```

The variable `response` seems to be unnecessary, am I right?

Reply:
The variable may be removed, but we still need `await responsePromise;`